vernac/metasyntax.mli

```diff
@@ -24,6 +24,11 @@ val add_notation : locality_flag -> constr_expr ->
 
 val add_notation_extra_printing_rule : string -> string -> string -> unit
 
+(** Fine-grained activation and deactivation of notations *)
+
+val notation_activation :
```

Comment:
Moving the code interpreting the commands to metasyntax was asked by @herbelin.

Defining a record is much heavier than these two option types and their meaning is quite clear from the code in metasyntax.  So I am not sure about the benefit of switching to records.